.github/workflows/pull-request.yml

```
@@ -0,0 +1,666 @@
+  Check-ESlint-Disable:
+    name: Check for eslint-disable
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0  # Fetch all history for all branches and tags
+
+      - name: Get changed files
+        id: changed-files
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:5:62: Double quote to prevent globbing and word splitting

(shellcheck)